client/styles/components/_p5-light-codemirror-theme.scss

```diff
@@ -12,7 +12,7 @@
 
```

Comment:
These changes to the syntax highlighting should be in a separate pull request, if possible.

Reply:
I think I committed these changes again. These were part of a separate PR which is already merged.

Reply:
I think these changes didn't get added to that PR though, since if they did they wouldn't be coming up in the diff.

Reply:
Then I will create a separate PR and commit these changes.